checkbox-provider-ce-oem/bin/graphics_test.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+test_ubuntu_frame_launching() {
+    if pgrep -if "ubuntu-frame" > /dev/null; then
+        echo "The ubuntu-frame is active"
+        echo "No need to bring it up again"
+        echo
+        echo "journal log of ubuntu frame:"
+        journalctl -b 0 | grep "ubuntu-frame"
+        touch "$PLAINBOX_SESSION_SHARE/ubuntu-frame-is-activating"
```

Comment:
we could simply grep journal log by `journalctl -b 0 -g "ubuntu-frame".
another issue is are we going to dump the journal log into 'ubuntu-frame-is-activating'?